test/specs/utils/cryptoModule.js

```diff
@@ -241,5 +241,35 @@ describe('Crypto class', () => {
 				});
 			},
 		);
+		Given(
```

Comment:
We should have a `describe` block for `#verifyMessage`. Can we nest it under the `Given('a message "Hello Lisker"')` step alongside `#signMessage`? It would involve breaking this step into parts.